src/Machete/DateTimeExtensions.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace Machete
+{
+    using System;
+
+
+    public static class DateTimeExtensions

```

Comment:
Extensions to built-in types are discouraged, as they create noise in the namespace. An extension method on the `Value<DateTimeOffset` would be more appropriate.